packages/std/src/math/decimal.rs

```diff
@@ -553,6 +561,28 @@ mod tests {
         assert_eq!(left * right, Uint128(0));
     }
 
+    #[test]
+    // in this test the Decimal is on the left
```

Comment:
Is this comment intentional? Looks like a copy/paste error. I used it because `a * b` and `b * a` are commutative but use different implementations in Rust.